packages/repository-tests/src/crud/relations/acceptance/belongs-to.relation.acceptance.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect, toJSON} from '@loopback/testlab';
+import {
+  deleteAllModelsInDefaultDataSource,
+  withCrudCtx,
+} from '../../../helpers.repository-tests';
+import {
+  CrudFeatures,
+  CrudRepositoryCtor,
+  CrudTestContext,
+  DataSourceOptions,
+} from '../../../types.repository-tests';
+import {Customer, Order, Shipment} from '../fixtures/models';
+import {givenBoundCrudRepositories} from '../helpers';
+
+export function belongsToRelationAcceptance(
+  dataSourceOptions: DataSourceOptions,
+  repositoryClass: CrudRepositoryCtor,
+  features: CrudFeatures,
+) {
+  describe('BelongsTo relation (acceptance)', () => {
+    before(deleteAllModelsInDefaultDataSource);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let customerRepo: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let orderRepo: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let shipmentRepo: any;
+
+    before(
+      withCrudCtx(async function setupRepository(ctx: CrudTestContext) {
+        ({customerRepo, orderRepo, shipmentRepo} = givenBoundCrudRepositories(
+          ctx.dataSource,
+          repositoryClass,
+        ));
+        const models = [Customer, Order, Shipment];
+        await ctx.dataSource.automigrate(models.map(m => m.name));
+      }),
+    );
+
+    beforeEach(async () => {
+      await orderRepo.deleteAll();
+    });
+
+    it('can find customer of order', async () => {
+      const customer = await customerRepo.create({
+        name: 'Order McForder',
+      });
+      const order = await orderRepo.create({
+        customerId: customer.id,
+        description: 'Order from Order McForder, the hoarder of Mordor',
+      });
+
+      const result = await orderRepo.customer(order.id);
+      // don't need to check parentId at this point, but still need to pass it
+      // in here so that MySQL won't complain
```

Comment:
Interesting. I'd like to better understand what's going under the hood. Which model instance (`customer` vs. `result`) is missing `parentId` property? Maybe we need to fix the implementation of our relation repositories to add FK property?

Anyhow, I guess this can be fixed later, once this PR is done. Can you add `FIXME` keyword to the comment please (to make it easier to find) and perhaps open a follow-up issue to investigate more?

Reply:
We pass in features.emptyValue to the `customer` now it should be fine.